README.md

```diff
@@ -2,4 +2,4 @@
 
 It's some delicious Javascript to do things with.
 
-No one's going to see the readme, huh?
+No one's going to see the readme, huh?
```

Comment:
If you want to change the README, absolutely feel free to, I don't know why I left it as this.